test/react-native/scripts/build-android-locally.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+const common = require('../../../scripts/common.js')
+const helper = require('../../../scripts/react-native-helper.js')
+
+// Check for required environment variables
+const requiredVars = ['REACT_NATIVE_VERSION', 'REGISTRY_URL']
+let allPresent = true
+requiredVars.forEach((envvar) => {
+  if (!process.env[envvar]) {
+    console.error(`Environment ${envvar} must be set`)
+    allPresent = false
+  }
+})
+if (!allPresent) return
+
+common.run('rm -rf docker-temp')
+common.run('mkdir -p docker-temp/fixtures')
```

Comment:
can you add this path to `.gitignore`, or put it in a system-wide tempdir so that it gets cleaned up by the OS?

Node docs for [`os.tmpdir()`](https://nodejs.org/dist/latest-v12.x/docs/api/os.html#os_os_tmpdir) and [`fs.mkdtemp()`](https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_fs_mkdtemp_prefix_options_callback) if you want to go the system route.

Reply:
I've already added it to `.gitignore`, but we might find the need for it to disappear altogether in future iterations.  I'm not particular fond of the temporary directory myself, but if we can keep it as is and I'll either remove it or change to an OS tempdir next time around if possible.